index.js

```diff
@@ -126,9 +137,10 @@ let rcl = {
   /**
   * Init the module.
   * @param {Context} context - The context, default is Context.defaultContext().
+  * @param {String[]} argv - Process commandline arguments.
```

Comment:
nit: should be lowercase 's' on string. Idk if jsdoc distinguishes between 'string' and 'String'

Reply:
fixed in 9e4b732

Reply:
fixed in 9e4b732